tools/generate_events.py

```diff
@@ -34,8 +34,17 @@
 type {{eventName}} struct {
     eventName string
 {%- for p, t in event.properties.items() %}
+    {%- if event.type != "host" %}
     {{event.pascal_case(p)}} {{event.go_type(t)}}
+    {%- else -%}
+    {%- if event.pascal_case(p) != "Info" %}
+    {{event.pascal_case(p)}} {{event.go_type(t)}}
+    {%- endif %}
+    {%- endif %}
 {%- endfor %}
+    {%- if event.type == "host" %}
+    Info string
```

Comment:
I think the right place to add this would be in the event definition file [here](https://github.com/openshift/assisted-service/blob/e781cea7e66fb4a75537961d5569a917491be8fd/docs/events.yaml).

That said, it makes me wonder if it's worth adding a currently unused field to all host event types seemingly just to be able to test this patch.

Will have to think about this a bit more and may want some input from @nmagnezi